tests/test_kernelshap.py

```diff
@@ -1,6 +1,6 @@
 from unittest import TestCase
 import numpy as np
-from dianna.methods.kernelshap import KERNELSHAPImage
+from dianna.methods.kernelshap_image import KERNELSHAPImage
```

Comment:
I found out why we had this all capital naming. It is because of the way that a method is fetched through the interface:
https://github.com/dianna-ai/dianna/blob/d788cd787cded1aeb5b17a9566744b43728fb5e0/dianna/__init__.py#L115-L117

The `method` argument will be converted to capital, which is consistent across all methods. 

We can change it to PascalCase later, in another PR (e.g. we could rename it as "SHAPImage").